### PR TITLE
790 ac multiplex participation

### PR DESCRIPTION
**Pull request by annacanal**
- [x] multiplex participation
- [x] average multiplex participation
- [x] Overlapping Degree
- [x] multiplex in-participation
- [x] Overlapping In-Degree
- [x] multiplex out-participation
- [x] Overlapping Out-Degree

### CODE REVIEW

File: braph2genesis/measures/_MultiplexInParticipation.gen.m [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+%%% ¡prop!
+M (result, cell) is the multiplex in-participation.
+%%%% ¡calculate!
+g = m.get('G');  % graph from measure class
+L = g.layernumber();
+
+if L == 0
+    value = {};
+else
+    B = g.get('A');  % cell with adjacency matrix (for graph) or 2D-cell array (for multigraph, multiplex, etc.)
```
Comment (c-yuwei) [inline braph2genesis/measures/_MultiplexInParticipation.gen.m:32]: is b not being used here...?

File: braph2genesis/measures/_MultiplexParticipationAv.gen.m [hunk 2]
```diff
@@ -0,0 +1,81 @@
+%% ¡header!
+MultiplexParticipationAv < MultiplexParticipation (m, average multiplex participation) is the graph average multiplex participation.
+
+%%% ¡description!
+The multiplex participation of a graph is the average heterogeneity
```
Comment (c-yuwei) [inline braph2genesis/measures/_MultiplexParticipationAv.gen.m:5]: should be "The average multiplex ....." ?

Comment (annacanal) [reply braph2genesis/measures/_MultiplexParticipationAv.gen.m]: yes. updated ^^

**Review by egolol — APPROVED** [review]

File: braph2genesis/measures/_MultiplexInParticipation.gen.m (hunk 1 above)
Comment (annacanal) [reply braph2genesis/measures/_MultiplexInParticipation.gen.m:32]: no .deleted

**Review by c-yuwei — APPROVED** [review]

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**